### ISSUE TITLE
Build error from parking_lot_core crate on linux x86_64

### ISSUE THREAD

**Issue by AaronM04** [rust-windowing/winit#1115]
I ran into a build error while building ggez, which has winit as a dependency:

```
   Compiling parking_lot_core v0.5.0
   Compiling gfx_device_gl v0.16.1
   Compiling crossbeam-deque v0.7.1
   Compiling lyon_path v0.13.2
error[E0432]: unresolved import `crate::thread_parker`
 --> /home/x/.cargo/registry/src/github.com-1ecc6299db9ec823/parking_lot_core-0.5.0/src/parking_lot.rs:8:12
  |
8 | use crate::thread_parker::ThreadParker;
  |            ^^^^^^^^^^^^^ maybe a missing `extern crate thread_parker;`?

error[E0432]: unresolved import `crate::thread_parker`
 --> /home/x/.cargo/registry/src/github.com-1ecc6299db9ec823/parking_lot_core-0.5.0/src/spinwait.rs:8:5
  |
8 | use crate::thread_parker;
  |     ^^^^^^^^^^^^^^^^^^^^ no `thread_parker` in the root

error[E0432]: unresolved import `crate::thread_parker`
 --> /home/x/.cargo/registry/src/github.com-1ecc6299db9ec823/parking_lot_core-0.5.0/src/word_lock.rs:9:12
  |
9 | use crate::thread_parker::ThreadParker;
  |            ^^^^^^^^^^^^^ maybe a missing `extern crate thread_parker;`?

   Compiling ggez v0.5.1 (/home/x/dev/rust/life/ggez)
error: cannot find macro `__cfg_if_items!` in this scope
  --> /home/x/.cargo/registry/src/github.com-1ecc6299db9ec823/parking_lot_core-0.5.0/src/lib.rs:61:1
   |
61 | / cfg_if! {
62 | |     if #[cfg(all(has_sized_atomics, target_os = "linux"))] {
63 | |         #[path = "thread_parker/linux.rs"]
64 | |         mod thread_parker;
...  |
90 | |     }
91 | | }
   | |_^
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0432`.
error: Could not compile `parking_lot_core`.
warning: build failed, waiting for other jobs to finish...
```

Partial output of `cargo tree`, showing relevant crates:
```
│       └── winit v0.19.2
│           ├── lazy_static v1.3.0 (*)
│           ├── libc v0.2.62 (*)
│           ├── log v0.4.8 (*)
│           ├── parking_lot v0.8.0
│           │   ├── lock_api v0.2.0
│           │   │   └── scopeguard v1.0.0
│           │   └── parking_lot_core v0.5.0
```

Rust version:
```$ rustc --version
rustc 1.37.0 (eae3437df 2019-08-13)
```

**Comment by AaronM04** [comment]
Confirmed! Here's the new `cargo tree` excerpt after updating to `0.19.3`:

```
│       └── winit v0.19.3
│           ├── lazy_static v1.3.0 (*)
│           ├── libc v0.2.62 (*)
│           ├── log v0.4.8 (*)
│           ├── parking_lot v0.9.0
│           │   ├── lock_api v0.3.1
│           │   │   └── scopeguard v1.0.0
│           │   └── parking_lot_core v0.6.2
│           │       ├── cfg-if v0.1.9 (*)
│           │       ├── libc v0.2.62 (*)
│           │       └── smallvec v0.6.10
```

It compiles fine.

**Comment by AaronM04** [comment]
Fixed in 0.19.3.
